rosbag2_py/test/test_sequential_reader.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+from rosidl_runtime_py.utilities import get_message
+
+from rclpy.serialization import deserialize_message
+
+from std_msgs.msg import String
+from rcl_interfaces.msg import Log
+
+from common import get_rosbag_options
+
+import rosbag2_py._rosbag2_py as rosbag2_py
+
+def test_sequential_reader():
+
+  bag_path = os.path.join(
```

Comment:
nitpick: we use four space indentation for Python (I think almost everywhere) in ROS 2.

I'm a big surprised that the linters don't care.

Reply:
The linters do care :) - but the flake8 check is temporarily disabled in the Github Actions

Reply:
Adding the common linters in package.xml caught them.
d261c63